build.go

```diff
@@ -136,7 +272,8 @@ Environment Variables:
 	}
 	BUILDDATE := time.Now().UTC().Format(time.RFC3339)
 
-	GOVERSION := cmd("go", "version")
+	GOVERSION := runtime.Version() //cmd("go", "version")
```

Comment:
No need to keep comments of old lines imo. Git keeps those in the commit history for us. :)